libs/SalesforceSDK/src/com/salesforce/androidsdk/app/SalesforceSDKManager.kt

```diff
@@ -221,6 +238,10 @@ open class SalesforceSDKManager protected constructor(
         LoginServerManager(appContext)
     }
 
+    /** The Salesforce SDK manager's native login manager */
+    lateinit var nativeLoginManager: NativeLoginManagerInterface
```

Comment:
Wouldn't this be safer as an optional rather than a `lateinit`? Since this is public to apps, they could get themselves into a crash if they don't remember to check if this is initialized.  `lateinit` is neat to have, but I've seen enough random crashes leak through that I almost never use it even privately anymore.

Reply:
Re-reading my own notes thus far, they're all optional except this one.  I do feel we'd save some apps an otherwise obvious crash or two by making this an optional.

Reply:
Perhaps.  That seems like a setup issue that customers wouldn't actually see in the wild (similar to a comment I made on your PR earlier).  I feel like customers are either going to use the feature and call `useNativeLogin` in the app's `onCreate` or they aren't going to touch `nativeLoginManager`.  Dealing with optionals is pretty trivial in Kotlin but a bit more annoying in Java.  🤷 

Thoughts on this @wmathurin ?

Reply:
Optionals are annoying but compile time failure is still preferable to a run time one (even if unlikely).